pkg/agent/nodeportlocal/npl_agent_test.go

```diff
@@ -673,26 +808,31 @@ func TestNodePortAlreadyBoundTo(t *testing.T) {
 	var nodePort int
 	testConfig := newTestConfig().withCustomPortOpenerExpectations(func(mockPortOpener *portcachetesting.MockLocalPortOpener) {
 		gomock.InOrder(
-			mockPortOpener.EXPECT().OpenLocalPort(gomock.Any()).Return(nil, portTakenError),
-			mockPortOpener.EXPECT().OpenLocalPort(gomock.Any()).DoAndReturn(func(port int) (portcache.Closeable, error) {
+			mockPortOpener.EXPECT().OpenLocalPort(gomock.Any(), gomock.Any()).Return(nil, portTakenError),
+			mockPortOpener.EXPECT().OpenLocalPort(gomock.Any(), gomock.Any()).DoAndReturn(func(port int, protocol string) (portcache.Closeable, error) {
```

Comment:
nit: we could use "tcp" here for the protocol instead of `gomock.Any()` since we know what it is